tests/sentry/web/frontend/test_js_sdk_loader.py

```diff
@@ -104,6 +109,86 @@ def test_greater_than_v7_returns_es5(
         self.assertTemplateUsed(resp, "sentry/js-sdk-loader.js.tmpl")
         assert b"/8.3.15/bundle.es5.min.js" in resp.content
 
+    @mock.patch("sentry.loader.browsersdkversion.load_version_from_file", return_value=["7.37.0"])
+    @mock.patch(
+        "sentry.loader.browsersdkversion.get_selected_browser_sdk_version", return_value="latest"
+    )
+    def test_bundle_kind_modifiers(self, load_version_from_file, get_selected_browser_sdk_version):
+        settings.JS_SDK_LOADER_DEFAULT_SDK_URL = "https://browser.sentry-cdn.com/%s/bundle%s.min.js"
+        settings.JS_SDK_LOADER_SDK_VERSION = "7.32.0"
+
+        for data, expected in [
```

Comment:
couldn't make `@mock.patch` and `pytest.mark.parametrize` play nicely with each other, so elected for the manual for loop.

Reply:
Yeah pytest.parametrize and unittest.TestCase don't play nice either.